handler/handler.go

```diff
@@ -54,94 +54,48 @@ func (batchHandler BatchHandler) sendToES(ctx context.Context, esDestURL string,
 
 	log.Info(ctx, "bulk events into ES starts")
 	target := len(events)
-	documentList := make(map[string]models.SearchDataImportModel)
 
-	var bulkcreate []byte
+	var bulkupsert []byte
 	for _, event := range events {
-		if event.Title == "" {
```

Comment:
Is there any reason why this validation is dismissed? 
And should there be any other validations in place?

Reply:
reintroduced using uid instead